run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+python setup.py develop
```

Comment:
what does the arg `develop` do?

Reply:
to set up the dependencies so we can run the app, I think. Also, for actual deployment, we should use install, but I am not sure on this yet. However, for it to run for demo purpose, develop will be sufficient.